modules/health_quest/config/rsa/sandbox_rsa

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Should we be keeping these keys external to the repo and pulling them in during deploy from credstash or similar sensitive storage system?

Reply:
Sounds like the safe approach. What is credstash and how do I use it?(I'm new)

Reply:
Reach out to the ops support engineer. He'll be able to guide you.

You'll probably want to rotate that key since it's stored in git now. You can send the key to the engineer over Keybase and they can add it to the devops repo which will drop the file into place upon deploy.

Reply:
This is a public repo, so this key is no longer private.